packages/data-stores/src/site/actions.ts

```diff
@@ -398,7 +398,8 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		}
 
 		const themeSetupOptions: ThemeSetupOptions = {
-			trim_content: true,
+			trim_content: options?.trimContent ?? true,
+			posts_source_site_id: options?.posts_source_site_id,
```

Comment:
[nit] We can set the `posts_source_site_id` property when it's provided, and then we don't need to update the test case.